PID:
  Kp_x : 2.0
  Kp_y : 2.0
  Kp_z : 2.6
  Kp_vx : 0.4
  Kp_vy : -0.4
  Kp_vz : 0.1

  Ki_x : 0.01
  Ki_y : 0.01                      
  Ki_z : 0.02
  Ki_vx : 0.0
  Ki_vy : 0.0
  Ki_vz : 0.1

  Kd_x : 0
  Kd_y : 0
  Kd_z : 0
  Kd_vx : 0
  Kd_vy : 0
  Kd_vz : 0
  K_yaw : 1.0

  position_x_i_bound : 1.0
  position_y_i_bound : 1.0
  position_z_i_bound : 5.0
  velocity_x_i_bound : 0.8
  velocity_y_i_bound : 0.8
  velocity_z_i_bound : 0.8
  thrust_bound : 0.36
  x_bound : 0.4
  y_bound : 0.4
  hover_height : 0.4
  

# 单环PID混合控制
PID_single:
  Kpx_single : 0
  Kpy_single : 0
  Kpz_single : 0
  Kvx_single : 0
  Kvy_single : 0
  Kvz_single: 0

Lowpass_filter:
  lp3_k : 0.1

Drone:
  mass : 0.305
  g : 9.80665
  balance_thrust : 0.3
